biz.aQute.bndlib.tests/test/test/annotationheaders/StdAnnotationHeadersTest.java

```
@@ -367,4 +368,59 @@ public void testStdRepeatableMetaAnnotated() throws Exception {
 		}
 	}
 
+	/**
+	 * Override bundle annotation processing with the `-bundleannotations`
+	 * instruction
+	 *
+	 * @throws Exception
+	 */
+	public void testOverrideBundleAnnotationsInstruction_failure() throws Exception {
+		try (Builder b = new Builder()) {
+			b.addClasspath(IO.getFile("bin_test"));
+			b.setPrivatePackage("test.annotationheaders.attrs.std.activator");
+			b.setProperty(Constants.BUNDLE_ACTIVATOR,
+				"test.annotationheaders.attrs.std.activator.TypeInVersionedPackage2");
+			b.setProperty("-bundleannotations", "*");
+			b.setProperty("-upto", "4.1");
+			b.build();
+			assertTrue(b.check());
+			Manifest manifest = b.getJar()
+				.getManifest();
+			manifest.write(System.out);
+
+			Attributes mainAttributes = manifest.getMainAttributes();
+
+			Header repeated = Header.parseHeader(mainAttributes.getValue(Constants.BUNDLE_ACTIVATOR));
+			assertThat(repeated).containsOnlyKeys("test.annotationheaders.attrs.std.activator.TypeInVersionedPackage");
+			fail("Should have already failed with two Bundle-Activator values");
```

This fail will be eaten by the empty catch!